package.json

```
@@ -118,7 +118,7 @@
     "semver": "^6.0.0",
     "snyk-config": "4.0.0",
     "snyk-cpp-plugin": "2.2.1",
-    "snyk-docker-plugin": "4.19.3",
+    "snyk-docker-plugin": "^4.20.1",
```

If we would like to be more strict about the internal dependencies and are not fine if users install latest minor version of a package we might want to remove `^` (atm with caret we can install e.g. versions 4.32.0 if that's the latest minor version which might not be what we want)
```suggestion
    "snyk-docker-plugin": "4.20.1",
```

---

Thanks! I removed the ^

---

Regarding package-lock.json, I have to admit that I'm not sure why it's not being used in this repo.
I'm sure there is a good explanation for that, and would love to know what it is..
Maybe @Avishagp could explain this?

---

This is done in order to have the most similar dependency tree during test runs, as it would be after installing snyk as a library. Done in this PR:
https://github.com/snyk/snyk/pull/223
If you need any further explanation let me know :)

---

thanks @Avishagp for explaining, so I assume we expect users to use `npm install` as opposed to `npm ci` when installing snyk cli which is fair 👍